frontend/src/container/AppLayout/index.tsx

```diff
@@ -71,7 +72,14 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 	const isPremiumChatSupportEnabled =
 		useFeatureFlags(FeatureKeys.PREMIUM_SUPPORT)?.active || false;
 
+	const isChatSupportEnabled =
+		useFeatureFlags(FeatureKeys.CHAT_SUPPORT)?.active || false;
+
+	const isCloudUserVal = isCloudUser();
+
 	const showAddCreditCardModal =
```

Comment:
the same check is present at multiple places we should add everywhere ?

Reply:
Whats the expected behaviour here?

Reply:
not needed. resolving